templates/mustache-templates/teamsBot.ts.mustache

```diff
@@ -65,25 +69,29 @@ export class {{#IS_BOT}}TeamsBot{{/IS_BOT}}{{^IS_BOT}}MessageExtensionBot{{/IS_B
     });
   }
 
-  async run(context: TurnContext) {
-    await super.run(context);
-
-    // Save any state changes. The load happened during the execution of the Dialog.
-    await this.conversationState.saveChanges(context, false);
-    await this.userState.saveChanges(context, false);
-  }
-
-  async handleTeamsSigninVerifyState(context: TurnContext, query: SigninStateVerificationQuery) {
-    console.log("Running dialog with signin/verifystate from an Invoke Activity.");
-    await this.dialog.run(context, this.dialogState);
-  }
-
-  async handleTeamsSigninTokenExchange(context: TurnContext, query: SigninStateVerificationQuery) {
-    await this.dialog.run(context, this.dialogState);
+  // Invoked when the bot is sent an Adaptive Card Action Execute.
+  async onAdaptiveCardInvoke(
+    context: TurnContext,
+    invokeValue: AdaptiveCardInvokeValue
+  ): Promise<AdaptiveCardInvokeResponse> {
+    if (invokeValue.action.verb === "userlike") {
```

Comment:
I would add a comment like this:
// The verb `userlike` is sent from the Adaptive Card defined in welcome.json

Reply:
// The verb "userlike" is sent from the Adaptive Card defined in adaptiveCards/welcome.json